packages/uniforms-material/src/ListField.js

```diff
@@ -1,46 +1,53 @@
-import React                  from 'react';
-import Subheader              from 'material-ui/Subheader';
-import connectField           from 'uniforms/connectField';
-import filterDOMProps         from 'uniforms/filterDOMProps';
-import joinName               from 'uniforms/joinName';
-import {Children}             from 'react';
-import {List as ListMaterial} from 'material-ui/List';
+import connectField   from 'uniforms/connectField';
+import filterDOMProps from 'uniforms/filterDOMProps';
+import joinName       from 'uniforms/joinName';
+import List           from '@material-ui/core/List';
+import ListSubheader  from '@material-ui/core/ListSubheader';
+import React          from 'react';
+import {Children}     from 'react';
+import {Fragment}     from 'react';
 
 import ListAddField  from './ListAddField';
 import ListItemField from './ListItemField';
 
-const List = ({
-    actionsStyle,
+const List_ = ({
     children,
+    dense,
     initialCount,
     itemProps,
     label,
     name,
     value,
     ...props
 }) =>
-    <ListMaterial {...filterDOMProps(props)}>
-        {!!label && <Subheader children={label} style={{paddingLeft: 0}} />}
-
-        {children ? (
-            value.map((item, index) =>
-                Children.map(children, child =>
-                    React.cloneElement(child, {
-                        key: index,
-                        label: null,
-                        name: joinName(name, child.props.name && child.props.name.replace('$', index))
-                    })
+    <Fragment>
+        <List
+            dense={dense}
+            subheader={label ? <ListSubheader disableSticky>{label}</ListSubheader> : undefined}
+            {...filterDOMProps(props)}
+        >
+            {children ? (
+                value.map((item, index) =>
+                    Children.map(children, child =>
+                        React.cloneElement(child, {
+                            key: index,
+                            label: null,
+                            name: joinName(name, child.props.name && child.props.name.replace('$', index))
+                        })
+                    )
+                )
+            ) : (
+                value.map((item, index) =>
+                    <ListItemField key={index} label={null} name={joinName(name, index)} {...itemProps} />
                 )
-            )
-        ) : (
-            value.map((item, index) =>
-                <ListItemField key={index} label={null} name={joinName(name, index)} {...itemProps} />
-            )
-        )}
-        <div style={{paddingTop: 8, paddingBottom: 8, ...actionsStyle}}>
-            <ListAddField name={`${name}.$`} initialCount={initialCount} />
-        </div>
-    </ListMaterial>
+            )}
+        </List>
+        <ListAddField name={`${name}.$`} initialCount={initialCount} />
+    </Fragment>
```

Comment:
If `@material-ui/core` doesn't require `react@16`, I'd rather go with the `<div />` and have more compatibility.